fix_arrow_tests.py

```
@@ -0,0 +1,9 @@
+import sys
+
+with open("crates/logfwd-output/src/arrow_ipc_sink.rs", "r") as f:
```

Ruff (0.15.11)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)